db/schema.rb

```diff
@@ -411,7 +413,7 @@
   end
 
   create_table "versions", force: :cascade do |t|
-    t.string "item_type", null: false
+    t.string "item_type"
```

Comment:
how did this go away?

Reply:
This was removed by hand, I got confused on whether or not it should be there lol